docs/user_guide/3_getting_started.md

```diff
@@ -178,11 +178,194 @@ When running the one-liner, along with the icon that starts the Grasper, you wil
+                               False: There are no un-checked-in modifications to this firmware. This should never be true.
+============================   ==============================================================================================
+```
+
+### Controller tuner
+It is possible to adjust the settings for any of the Position or Force (Motor) controllers.
+	**Plugins → Shadow Robot → Basic → Controller Tuner**
+#### Position controller
+  ![adjust_position_controller](../img/adjust_position_controller.png)
+
+
+
+  Here you can select a finger, thumb or wrist joints, and adjust the different position control parameters. Click ```Set Selected``` to send the new values to the motors and make them take effect.
+
+* **“P”, “I” & “D” terms:**  Gains parameter of the position PID controller. By default, Shadow tunes the paramenters using P or PD combinations. The user can add “I” gains in the control if he considers necessary.
+
+* **Max_force:** This puts a limit on the output (PWM) value that will be sent from the host to the motor by the position controller. It can be useful when setting up a controller for the first time to limit the motor power to a safe level.
+
+* **Position_Deadband:** The error is considered to be zero if it is within ±deadband. This value should be set as a little more than the noise on the sensor. The units of deadband are the same as the value being controlled. So, the deadband for a position controller is in radians.
+
+#### Force controller
+![adjust_torque_controller](../img/adjust_torque_controller.png)
+
+* **“P”, “I” & “D” terms:** Gains parameter of the torque PID controller. By default, Shadow tunes the paramenters using just P gain for the torque control.
+
+* **Max_PWM:** This puts a limit on the final PMW value that will be sent to the motor by the torque controller. It can be useful when setting up a controller for the first time to limit the motor power to a safe level.
+
+* **Deadband:** The error is considered to be zero if it is within ±deadband. This value should be set as a little more than the noise on the sensor. The units of deadband are the same as the value being controlled. The deadband for a torgue controller is in the units of the strain gauges.
+
+* **Torque_Limit:** This value is used to limit the PWM at the end of the control loop. The control algoritm reduces the final PWM that goes to the motor making sure that the force in the strain gauge doesn’t overcome this limit value.
+
+Click ```Save``` to save your settings.
+
+### Bootloader
+The firmware in the motors MCUs can be updated from the PC, without opening up the motor base. This can be done from the GUI. Shadow will send you a new HEX if there is an update.
+	**Plugins → Shadow Robot → Advanced → Motor Bootloader**
+
+You will see a window listing each motor board, along with its current firmware SVN revision number.
+
+![bootload_new_firmware](../img/bootloading_new_firmware.png)
```

Comment:
In the world document, there are captions for the figures. Could you check if there is an easy way to add them here too.